### PR TITLE
Answer

### PR DESCRIPTION
**Pull request by wesleyjaboin**

### CODE REVIEW

File: QUESTIONS.md [hunk 1]
```diff
@@ -3,3 +3,8 @@
 What questions can you come up with?
 
 ## Cohort Questions
+question1:
+Most web dev job posting required candidates to have a bachelor in CS. what about those of us who don not have a CS bachelor?
```
Comment (Emeralds99) [inline QUESTIONS.md]: Don't worry about it, apply anyway. This course is meant to make up for that difference in experience.

**Review by Emeralds99 — APPROVED** [review]
👍

File: QUESTIONS.md (hunk 1 above)
Comment (starlit79) [inline QUESTIONS.md]: I wonder about that sometimes too

File: QUESTIONS.md [hunk 2]
```diff
@@ -3,3 +3,8 @@
 What questions can you come up with?
 
 ## Cohort Questions
+question1:
```
Comment (jessicareynolds) [inline QUESTIONS.md]: Great question! I wonder if we would benefit from gaining more education experience.

File: QUESTIONS.md (hunk 1 above)
Comment (hollisjamison) [inline QUESTIONS.md]: Dont even worry about it. My last 3 jobs required bachelor degrees in the IT field.

That's the HR director writing that, if you talk to a senior developer or project manager they will only care about experience and your projects.

**Pull request merged**